src/main/java/com/jwplayer/southpaw/Southpaw.java

```diff
@@ -267,39 +267,43 @@ protected void build(int runTimeS) {
                     while (records.hasNext()) {
                         ConsumerRecord<BaseRecord, BaseRecord> newRecord = records.next();
                         ByteArray primaryKey = newRecord.key().toByteArray();
+
+                        logger.info("---------------------------------");
+                        logger.info(String.format("Processing %s record key: %s", entity, primaryKey.toString()));
```

Comment:
after your rebase you could replace with
```
logger.debug("Processing {} record key: {}", entity, primaryKey.toString());
```

Reply:
Removed it, was mostly to keep track of my changes when running it locally